src/components/pages/Projects.js

```diff
@@ -1,12 +1,99 @@
 import React from 'react';
 import '../../App.css';
-import Cards from '../Cards';
+import './Projects.css';
 
 export default function Projects() {
     return (
         <>
             <h1 className='projects'>PROJECTS</h1>
-            <Cards />
+            <div className='info_wrapper'>
+                <h5>CONSULTING AND 3D PRINTING</h5>
+                <p>Our main and most popular team activity! Starting at only $0.15/gram and a  
+                    $5 setup fee, our printing service is one of the most affordable on  campus! 
+                    Using our 6 available printers, we aim to offer low-cost, high quality 3D 
+                    printing services to UBC students, staff, and faculty. <br /><br />
```

Comment:
The correct syntax for breaks is <br> (the opening tag) and </br> (the closing tag).

Reply:
I tried changing the breaks but both <br></br> and just using <br /> did not work